### ISSUE TITLE
BUG - [SEV 3] - [Android] - [Allergy card transition to details does not announce title, lands in blank space]

### ISSUE THREAD

**Issue by b-rocha** [department-of-veterans-affairs/va-mobile-app#10534]
## Specs:

- Device: Pixel
- OS: Android
- User Account:
- Accessibility State: Talkback

## Steps to Reproduce

1. Go to the Allergies page, select an allergy to get to the Allergies Detail page
2. Upon going to the page, note that it does not  announce the title of page and lands in a blank space (i.e. the selection indicator lands in a blank area)

## Desired behavior

The screenreader should announce the title of the Allergies Details page; there should not be a blank "selectable" space present. 

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact:  Low 
- Frequency: High  

- 3 - Low

## Screen shot(s) and additional information

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, global, Health, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the relevant team's bug epic ([Global](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8302), [Health & Benefits](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8303), [API](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8304), [QART](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8305))
